core/worker.go

```diff
@@ -727,9 +736,6 @@ func (w *worker) GeneratePendingHeader(block *types.WorkObject, fill bool) (*typ
 
 	if nodeCtx == common.ZONE_CTX && w.hc.ProcessingState() {
 		work.batch.Reset()
-		for _, hash := range work.coinbaseLockupsCreated {
```

Comment:
so we not need this anymore, i remember this caused a bug previously

Reply:
Not needed anymore, using work.utxosCreated instead (for all muhash-related hash collection)